.github/workflows/deploy-docs.yml

```diff
@@ -12,11 +12,10 @@ jobs:
         uses: actions/checkout@v2
       - name: Build
         run: |
+          # Ensure all the top level pptr deps are installed
           npm install
-          # This will be replaced with the actual documentation builder once it's
-          # configured
-          mkdir -p docs-dist/
-          cp website/index.html docs-dist/index.html
+          npm run install-docusuarus-deps
```

Comment:
typo: "docusuarus"

Note: the fact we're using docusaurus is an implementation detail that could change in the future, so we might wanna use a generic name like `npm run install-doc-deps` or so. Same for the step below. (I could also imagine these as being just 1 step, since similarly, installing the nested deps is an implementation detail of the website build)